src/SFA.DAS.EAS.Portal.UnitTests/Application/Adapters/CohortAdapterTests.cs

```diff
@@ -0,0 +1,46 @@
+﻿using NUnit.Framework;
+using SFA.DAS.CommitmentsV2.Messages.Events;
+using SFA.DAS.EAS.Portal.Application.Adapters;
+using SFA.DAS.EAS.Portal.Worker.UnitTests.Builders;
+using FluentAssertions;
+using Moq;
+using NServiceBus;
+using System;
+
+namespace SFA.DAS.EAS.Portal.UnitTests.Application.Adapters
+{
+    [TestFixture]
```

Comment:
We should mark all our tests as `[Parallelizable]`, which has a huge positive impact on the time taken to run all unit tests on machines with many cores. If we create all new tests as parallizable, there is generally little cost to development time, but can be a pain to implement retrospectively.

Reply:
Good point.  Do they not run in parallel by default?

Reply:
This has been actioned